spec/rails_helper.rb

```diff
@@ -0,0 +1,40 @@
+
+ENV['RAILS_ENV'] ||= 'test'
+require File.expand_path('../../config/environment', __FILE__)
+
+abort("The Rails environment is running in production mode!") if Rails.env.production?
+require 'spec_helper'
+require 'rspec/rails'
+require 'database_cleaner'
+require 'simplecov'
+SimpleCov.start
+
+
+ActiveRecord::Migration.maintain_test_schema!
+
+RSpec.configure do |config|
+  config.fixture_path = "#{::Rails.root}/spec/fixtures"
+  DatabaseCleaner.strategy = :truncation
+  
+  config.use_transactional_fixtures = true
+
+  config.infer_spec_type_from_file_location!
+
+  config.filter_rails_from_backtrace!
+  
+  config.before(:all) do
+    DatabaseCleaner.clean
+  end
+
+  config.after(:each) do
+    DatabaseCleaner.clean
+  end
+
+end
+
```

Comment:
@RajaaBoulassouak This line needs to be added to make DB cleaner work properly. 

`DatabaseCleaner.strategy = :truncation`